components/server/src/authorization/relationship-updater.ts

```diff
@@ -53,41 +61,40 @@ export class RelationshipUpdater {
                 fromVersion: user?.additionalData?.fgaRelationshipsVersion,
                 toVersion: this.version,
             });
-            // TODO run in distributed lock
-            // return this.mutex.using([`fga-migration-${user.id}`], 2000, async () => {
-            const before = new Date().getTime();
+            return this.mutex.using([`fga-migration-${user.id}`], 2000, async () => {
```

Comment:
now that redis work in GHA build, we can again run this code in a distributed lock